.github/workflows/release.yml

```diff
@@ -1,16 +1,27 @@
-# TODO release XSD to timefold.ai/xsd/
-# TODO reset timefold-quickstarts stable branch
-# TODO adjust website pom.yml release date and version
-# TODO tag github issues with version
 name: Release
 on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'Release version'
+        description: 'Release version (e.g. 1.0.0)'
         required: true
       nextVersion:
-        description: 'Next version after release (-SNAPSHOT will be added automatically)'
+        description: 'Next version after release (e.g. 1.1.0, -SNAPSHOT will be added automatically)'
+        required: true
+      nextMicroVersion:
+        description: 'Next version after release for release branch (e.g. 1.0.1, -SNAPSHOT will be added automatically)'
+        required: true
+      sourceBranch:
+        description: 'Branch to cut the release from'
+        default: main
+        required: true
+      releaseBranch:
+        description: 'Release branch to create (e.g. 1.0.x for version 1.0.0; once created, branch protection rules apply)'
+        default: dry_run
+        required: true
```

Comment:
Let's move these two before the "Next version..." parameters to define these branches before referring to them.